tests/ts/liblastlog2/pam_lastlog2_output

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="pam_lastlog2_output"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+
+TESTPROG="$TS_HELPER_LIBLASTLOG2_PAM_LASTLOG2_OUTPUT"
+
+export TZ=UTC
+
+[ -x $TESTPROG ] || ts_skip "test not compiled"
+
+ts_run $TESTPROG >> $TS_OUTPUT.debug 2>&1
```

Comment:
`$TS_OUTPUT` will be compared to a reference output to make sure that the tests executes as expected.
`$TS_OUTPUT.debug` is not checked in this way.

In general the tests as they are do not assert anything.
Try to work with "$TS_OUTPUT" and "$TS_ERRLOG" if possible and do manual checks with `ts_fail` if that's not sufficient.